_documentation/bamboo.md

```diff
@@ -70,7 +70,7 @@ The tests are executed using dedicated accounts. These accounts are created by t
 
 If the former method is desired, then you should provide the credentials of an administrative user in the task configuration. When these credentials are not provided, then the latter method will be used to provision test accounts.
 
-Finally, you can provide a comma-separated list of tests that are to be skipped (For example: `EntityCapsTest,SoftwareInfoIntegrationTest`), or specifications (not case-sensitive) that are to be skipped (For example: `XEP-0045,XEP-0060`).
+Finally, you can control the tests that are to be attempted. You can provide a comma-separated list of tests that are to be skipped (For example: `EntityCapsTest,SoftwareInfoIntegrationTest`), or specifications (not case-sensitive) that are to be skipped (For example: `XEP-0045,XEP-0060`). Conversely, you can limit the run tests by providing a comma-separated list of tests or specifications (for example, for quick tests of refactored functionality, or to test functionality offered in a server with a plugin architecture).
```

Comment:
Not a big issue for me, but should we document what happens if `disable` and `enable` is combined? I think Smack has wording for this, somewhere.